content/blogs/2023-11-03-advanced-python-scripts.md

```diff
@@ -0,0 +1,328 @@
+---
+title: "Kestra and Python: How to Run and Integrate Complex Scripts into Your Flows"
+description: "Learn two recommended and scalable ways of including long and complex Python scripts into your Kestra flows."
+date: 2023-11-06T12:00:00
+category: Solutions
+author:
+  name: Dario Radecic
+  image: "dradecic"
+image:
+---
+
+When it comes to data orchestration platforms and Airflow alternatives, [Kestra](https://github.com/kestra-io/kestra) is a one-stop shop that allows you to integrate scripts from various programming languages, such as Python.
+
+The thing is, Python scripts with several hundreds of lines of code are messy in themselves. Things go nuclear when you paste that code into a YAML file, which is a data serialization language Kestra uses to organize flows.
```

Comment:
```suggestion
```

there's no need to mention that method, especially not by saying it's "nuclear" :) it works well for simple functions